src/ui/about_dialog.ui

```diff
@@ -12,7 +12,7 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>780</width>
+    <width>926</width>
```

Comment:
That's just the default resolution. It scales down if used on 800x600 automatically:

![workspace 1_018](https://cloud.githubusercontent.com/assets/110988/23341291/bd42ed52-fc44-11e6-9c6d-d3aa277c730b.png)